.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npm run lint
```

Comment:
This will take a long time to check the whole repo. Please use `lint-staged` instead. You can use [this example](https://github.com/aabounegm/cast/blob/main/.lintstagedrc.yaml) as a reference.